addons/sale_quotation_builder/models/product_template.py

```diff
@@ -17,8 +17,8 @@ class ProductTemplate(models.Model):
     def _compute_quotation_description(self):
         for record in self:
             if record.quotation_only_description:
-                record.quotation_description = record.quotation_only_description
+                record.quotation_description = record.with_context(lang=self.env.lang).quotation_only_description
```

Comment:
AFAIK, this doesn't change anything since you're putting as lang the existing environment language.

Reply:
> AFAIK, this doesn't change anything since you're putting as lang the existing environment language.

yes, changing this place seems a bit redundant